service/src/main/java/uk/nhs/adaptors/gp2gp/ehr/mapper/ImmunizationObservationStatementMapper.java

```diff
@@ -104,8 +107,7 @@ private String buildPertinentInformation(Immunization immunization) {
             buildDoseQuantityPertinentInformation(immunization),
             buildExplanationPertinentInformation(immunization),
             buildVaccinationProtocolPertinentInformation(immunization),
-            buildVaccineCode(immunization),
-            buildNotePertinentInformation(immunization)
+            buildVaccineCode(immunization)
```

Comment:
These are now in the wrong order. I think you should revert https://github.com/nhsconnect/integration-adaptor-gp2gp/pull/140/commits/83517109ffbe7f97a8e37bdc6e9eac45637cb4b5